src/Microsoft.DotNet.Configurer/CliFolderPathCalculator.cs

```diff
@@ -12,20 +12,23 @@ namespace Microsoft.DotNet.Configurer
 {
     public class CliFolderPathCalculator
     {
-        private const string ToolsFolderName = "tools";
+        // ToolsShimFolderName ToolPackageFolderName cannot be the same
+        // or if the PackageId is the same as CommandName, they will conflict on unix.
+        private const string ToolsShimFolderName = "tools";
+        private const string ToolPackageFolderName = "toolspkg";
```

Comment:
Nit: how about `toolpkgs`?  `toolspkg` seems a singular package to me.

Reply:
i feel it is better for them to have the same prefix. and "tools" are from nuget's folder /tools (beside /lib)

Reply:
also "pkg" as afraid of long path

Reply:
Compromise on `toolspkgs`? 😄 I can also live with this as is.

Reply:
done :)